project1/hello.cpp

```diff
@@ -0,0 +1,6 @@
+#include<iostream>
+using namespace std;
+int main()
+{
+ cout<<"Welcome"; 
```

Comment:
change to "Welcome World" and don't need to add separate commit. Keep only one commit per pr.